puppet/modules/secretsgit/manifests/init.pp

```diff
@@ -0,0 +1,22 @@
+class secretsgit(
+  String $group = 'secretsgit',
+  Stdlib::Absolutepath $path = '/srv/secretsgit',
+  Array[String] $users = []
+) {
+
+  group { $group:
+    ensure => present
+  }
+
+  file { $path:
+    ensure => directory,
+    owner => 'root',
+    group => $group,
+    mode => '2770',
+  }
+
+  $users.each |String $user| {
```

Comment:
I think we can also use `members => $users` on `group`.

Reply:
the [docs](https://puppet.com/docs/puppet/6.6/types/group.html#group-attribute-members) say "Requires features manages_members", and [`groupadd`](https://puppet.com/docs/puppet/6.6/types/group.html#group-provider-groupadd), the default implementation Puppet uses on Linux [does not have that feature](https://puppet.com/docs/puppet/6.6/types/group.html#group-provider-features).